beacon-chain/types/active_state.go

```diff
@@ -315,9 +312,8 @@ func (a *ActiveState) getSignedParentHashes(block *Block, attestation *pb.Aggreg
 	obliqueParentHashes := attestation.ObliqueParentHashes
 	earliestSlot := int(block.SlotNumber()) - len(recentBlockHashes)
 
-	startIdx := int(attestation.Slot) - earliestSlot - int(params.GetConfig().CycleLength) + 1
```

Comment:
The previous behavior was correct. I'll take a look at this to see what's going on.

Reply:
The original code is correct. `signedParentHashes` includes the attestation itself, which is why there's a +1. Was this causing an issue somewhere?